### PR TITLE
Reuse container definitions

### PR DESCRIPTION
**Pull request by jerith**

### CODE REVIEW

File: example/fixtures.py [hunk 1]
```diff
@@ -13,15 +13,11 @@
```
Comment (JayH5) [inline example/fixtures.py:13]: Should this still be called `fixtures.py` if we don't actually have any (pytest) fixtures in it? `resources.py`? (Maybe that is worse)

File: example/fixtures.py [hunk 2]
```diff
@@ -53,51 +49,42 @@ def for_fixture(
...
-beat_container = make_multi_container(
-    'beat_container', ['single_container', 'beat_only_container'])
-
-
-__all__ = [
-    'raw_db_container', 'db_container', 'raw_amqp_container',
-    'amqp_container', 'single_container', 'web_only_container',
-    'worker_only_container', 'beat_only_container', 'web_container',
-    'worker_container', 'beat_container']
+def make_combined_fixture(base):
```
Comment (JayH5) [inline example/fixtures.py]: Maybe a small comment about what's going on here or in `make_multi_fixture`?

Comment (jerith) [reply example/fixtures.py]: Will do. I'll probably just combine the two functions.

File: example/fixtures.py (hunk 1 above)
Comment (jerith) [reply example/fixtures.py:13]: Maybe `definitions.py`?

File: example/requirements-test.txt [hunk 3]
```diff
@@ -1,5 +1,6 @@
-seaworthy >= 0.1.1
+seaworthy >= 0.1.2
 iso8601
 pytest >= 3.1
 pytest-catchlog
+requests
```
Comment (JayH5) [inline example/requirements-test.txt]: Why was this added? I don't see it used anywhere?

**Review by JayH5 — APPROVED** [review]
:+1:

File: example/requirements-test.txt (hunk 3 above)
Comment (jerith) [reply example/requirements-test.txt]: That was accidental, I'll remove it. (It must have wandered in while I was looking through the revision history for the seaworthy git link.)
